### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Globulopolis**
Fixed https://github.com/exstreme/Jcomments-4/issues/60
Fixed https://github.com/exstreme/Jcomments-4/issues/58

### CODE REVIEW

File: component/administrator/language/en-GB/com_jcomments.ini [hunk 1]
```diff
@@ -255,7 +255,7 @@ AP_COMMENT_MINLENGTH="Minimum comment length"
 AP_COMMENT_MINLENGTH_DESC="The minimum number of characters a user should post. The user will get an alert if comment's length less then this value."
 AP_SHOW_COMMENTLENGTH="Character counter"
 AP_SHOW_COMMENTLENGTH_DESC="Enable/Disable displaying the character count while typing."
-AP_ENABLE_ACCEPT_TERMS_OF_USE="Don't show checkbox \"Accept terms of use\"."
+AP_ENABLE_ACCEPT_TERMS_OF_USE="Do not show the \"Accept terms of use\"."
```
Comment (ThomDietrich) [inline component/administrator/language/en-GB/com_jcomments.ini:258]: It is the other way around. Setting this option SHOWS the terms.

Therefore (beware typo in second): 
```
AP_ENABLE_ACCEPT_TERMS_OF_USE="Show \"Accept terms of use\" checkbox"
AP_ENABLE_ACCEPT_TERMS_OF_USE_DESC="Enable/Disable displaying the checkbox to accept terms of use."
```

Comment (Globulopolis) [reply component/administrator/language/en-GB/com_jcomments.ini:258]: Can you fix it?

File: component/site/tpl/default/tpl_comment.php [hunk 2]
```diff
@@ -42,12 +42,18 @@ public function render()
...
 				<div class="flex-fill border rounded rbox">
 					<?php
-					$commentBoxIndentStyle = ($this->getVar('avatar') == 1) ? ' avatar-indent' : '';
-
 					if ($this->getVar('avatar') == 1)
 					{
 						?>
-						<div class="comment-avatar"><?php echo $comment->avatar; ?></div>
+						<div class="comment-avatar">
+							<?php if ($comment->profileLink > ''): ?>
```
Comment (exstreme) [inline component/site/tpl/default/tpl_comment.php:49]: How you can compare non numeric values? Use !empty I think

File: component/administrator/language/en-GB/com_jcomments.ini (hunk 1 above)
Comment (exstreme) [reply component/administrator/language/en-GB/com_jcomments.ini]: https://github.com/exstreme/Jcomments-4/pull/61/commits/1ca19c839dc7fba886019f1a1d4f481d494cbb46 fixed

File: component/administrator/language/en-GB/com_jcomments.ini [hunk 3]
```diff
@@ -255,8 +255,8 @@ AP_COMMENT_MINLENGTH="Minimum comment length"
 AP_COMMENT_MINLENGTH_DESC="The minimum number of characters a user should post. The user will get an alert if comment's length less then this value."
 AP_SHOW_COMMENTLENGTH="Character counter"
 AP_SHOW_COMMENTLENGTH_DESC="Enable/Disable displaying the character count while typing."
-AP_ENABLE_ACCEPT_TERMS_OF_USE="Don't show checkbox \"Accept terms of use\"."
-AP_ENABLE_ACCEPT_TERMS_OF_USE_DESC="Enable/Disable displayint the checkbox to accept terms of use."
+AP_ENABLE_ACCEPT_TERMS_OF_USE="Show the \"Accept terms of use\"."
```
Comment (ThomDietrich) [inline component/administrator/language/en-GB/com_jcomments.ini:258]: Minus the dot, plus checkbox:

```suggestion
AP_ENABLE_ACCEPT_TERMS_OF_USE="Show the \"Accept terms of use\" checkbox"
```

**Pull request merged**